socorro/external/es/supersearch.py

```diff
@@ -481,7 +482,19 @@ def get(self, **kwargs):
                     hits.append(self.format_fields(hit.to_dict()))
 
                 total = search.count()
-                aggregations = self.format_aggregations(results.aggregations)
+
+                try:
+                    aggregations = self.format_aggregations(
+                        results.aggregations
+                    )
+                except AttributeError:
```

Comment:
Is this AttributeError for `results.aggregations`? Right now this gets used to capture that AND any potential `AttributeError` inside `self.format_aggregations`. I think it should be like this:

```python
try:
    aggregations = results.aggregations
except AttributeError:
    aggregations = None
if aggregations:
    aggregations = self.format_aggregations(aggregations)
```